server/log_rpc_server.go

```diff
@@ -37,6 +37,7 @@ const proofMaxBitLen = 64
 
 var (
 	optsLogRead            = trees.NewGetOpts(trees.Query, true, trillian.TreeType_LOG, trillian.TreeType_PREORDERED_LOG)
+	optsLogInit            = trees.NewGetOpts(trees.Queue, false, trillian.TreeType_LOG, trillian.TreeType_PREORDERED_LOG)
```

Comment:
trees.Admin perhaps?

Reply:
Yup, also fixed `optsPreorderedLogWrite` while I'm here.
I wonder if there is any difference between them apart from naming, seems to work both ways.